DirectProgramming/Fortran/guided_Coarray/Makefile

```diff
@@ -0,0 +1,50 @@
+## =============================================================
+## Copyright © 2020 Intel Corporation
+##
+## SPDX-License-Identifier: MIT
+## =============================================================
+##
+##
+##******************************************************************************
+## Content:
+##
+##  Build for guided_coarray
+##
+##  To compile and run the sequential version type: make seq
+##  To compile and run your coarray version type: make coarray
+##******************************************************************************
+
+FC=ifx
+FCFLAGS=
+FCFLAGS_COARRAY=-coarray
+
+SEQ=seq
+SEQ_OBJS= mcpi_sequential.o
+COARRAY=coarray
+COARRAY_OBJS= mcpi_coarray_final.o
+
+mcpi_sequential.o: ./src/mcpi_sequential.F90
+mcpi_coarray.o: ./src/mcpi_coarray_final.F90
+
+$(SEQ): $(SEQ_OBJS)
+	$(FC) $(SEQ_OBJS) -o $@
+	$(SEQ)
+
+$(COARRAY): $(COARRAY_OBJS)
+	$(FC) $(FCFLAGS_COARRAY) $(COARRAY_OBJS) -o $@
+	$(COARRAY)
```

Comment:
@krisdale Request changing line 35 from `$(COARRAY)` to `./$(COARRAY)` so that the executable will run from the current working directory.